drivers/client_location_history/app/models/grda_warehouse/hmis.rb

```diff
@@ -0,0 +1,11 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module ClientLocationHistory::GrdaWarehouse
+end
```

Comment:
Yes, but rails seemed very confused about where it wanted that module defined.  Since there's no logic in the model, it should do no harm to have it defined multiple times.